apps/docs/content/guides/database/connecting-to-postgres.mdx

```diff
@@ -26,384 +20,121 @@ Supabase provides auto-updating Data APIs. These are the easiest way to get star
-
-Find your connection settings. Go to your [`Database Settings`](https://supabase.com/dashboard/project/_/settings/database) and make sure `Use connection pooling` is checked. Change the connection mode to `Session`, and copy the parameters into the connection string:
-
-```shell
-psql "sslmode=verify-full sslrootcert=/path/to/prod-supabase.cer host=[CLOUD_PROVIDER]-0-[REGION].pooler.supabase.com dbname=postgres user=postgres.[PROJECT_REF]"
+```md
+// Example String
+postgres://postgres.apbkobhfnmcqqzqeeqss:[YOUR-PASSWORD]@aws-0-ca-central-1.pooler.supabase.com:5432/postgres
 ```
 
-### Connecting with Prisma
-
-Refer to our [Prisma integration guide](https://supabase.com/partners/integrations/prisma) for more details.
-
-### Connecting with Postgres.js
-
-[Postgres.js](https://github.com/porsager/postgres) is a full-featured PostgreSQL client for Node.js and Deno.
-
-<StepHikeCompact>
-
-  <StepHikeCompact.Step step={1}>
-
-    <StepHikeCompact.Details title="Install">
-
-    Install Postgres.js and related dependencies.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
-
-     ```shell
-     npm i postgres
-     ```
-
-    </StepHikeCompact.Code>
+### Transaction mode (port 6543):
 
-  </StepHikeCompact.Step>
-
-  <StepHikeCompact.Step step={2}>
-
-    <StepHikeCompact.Details title="Connect">
-
-    Create a `db.js` file with the connection details.
-
-    To get your connection details, go to your [`Database Settings`](https://supabase.com/dashboard/project/_/settings/database). Make sure `Use connection pooling` is enabled. Choose `Transaction Mode` if you're on a platform with transient connections, such as a serverless function, and `Session Mode` if you have a long-lived connection. Copy the URI and save it as the environment variable `DATABASE_URL`.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
-
-    ```ts
-    // db.js
-    import postgres from 'postgres'
-
-    const connectionString = process.env.DATABASE_URL
-    const sql = postgres(connectionString)
-
-    export default sql
-    ```
-
-    </StepHikeCompact.Code>
-
-  </StepHikeCompact.Step>
-
-  <StepHikeCompact.Step step={3}>
-
-    <StepHikeCompact.Details title="Execute commands">
-
-    Use the connection to execute commands.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
-
-    ```ts
-    import sql from './db.js'
-
-    async function getUsersOver(age) {
-      const users = await sql`
-        select name, age
-        from users
-        where age > ${ age }
-      `
-      // users = Result [{ name: "Walter", age: 80 }, { name: 'Murray', age: 68 }, ...]
-      return users
-    }
-    ```
-
-    </StepHikeCompact.Code>
-
-  </StepHikeCompact.Step>
-
-</StepHikeCompact>
+<Admonition type="caution">
+  Transaction mode does not support [prepared
+  statements](https://postgresql.org/docs/current/sql-prepare.html). To avoid errors, disable
+  prepared statements in your connection library.
```

Comment:
it is supported, but only with `PREPARED` statements :stuck_out_tongue_winking_eye: